docs/en/observability/apm/tab-widgets/install-agents.asciidoc

```diff
@@ -137,44 +125,55 @@ func main() {
 
 *Add the agent dependency to your project*
 
-Add the Elastic APM iOS Agent as a
-https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app[package dependency]
-to your Xcode project or your `Package.swift`:
+Add the Elastic APM iOS Agent to your Xcode project or your `Package.swift`.
+
+Here are instructions for adding a https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app[package dependency] to a standard Xcode project.
 
+Details of adding dependencies to your Package.swift can be found on https://developer.apple.com/documentation/xcode/creating_a_standalone_swift_package_with_xcode#3578941['Add a Dependency on Another Swift Package'].
+Below is a helpful code-snippet:
+
+`package.swift`:
 [source,swift,linenums,highlight=2;10]
 ----
 Package(
     dependencies:[
-        .package(name: "iOSAgent", url: "git@github.com:elastic/apm-agent-ios.git", .branch("main")),
+         .package(name: "apm-agent-ios", url: "https://github.com/elastic/apm-agent-ios.git", from: "1.0.0"),
     ],
   targets:[
     .target(
         name: "MyApp",
         dependencies: [
-            .product(name: "iOSAgent", package: "iOSAgent")
+            .product(name: "ElasticApm", package: "apm-agent-ios")
         ]
     ),
 ])
 ----
 
 *Initialize the agent*
 
-If you're using `SwiftUI` to build your app, add the following to `App.swift`:
+If you're using `SwiftUI` to build your app add the following to your `App.swift`:
 
-[source,swift,linenums,swift,highlight=2;7..12]
+[source,swift,linenums,swift,highlight=2;4..15;19]
```

Comment:
What does `highlight=2;4..15;19` do? I've never seen that before and I can't tell in the rendered result. 🤔

Reply:
How very interesting. Looks like this is supported by [Asciidoctor](https://docs.asciidoctor.org/asciidoc/latest/verbatim/highlight-lines/#highlight-attribute) but likely stripped out by our build at some point. I'm not sure why Bryce has added those. I can remove them.

Reply:
Fixed in b4f3240